### DISCUSSION TITLE
Which icon is used on the documentation website?

### DISCUSSION THREAD

**Discussion by mxvsh** [heroui-inc/heroui#1944] [Q&A]
I really liked the icons used on the documentation website in the demos. I see that they are imported from a file containing SVG. I wonder is there any specific names of these kinds of icons? Where can I get them. Thanks!

**Comment by iamharshdabas** [comment]
It appears that [Iconly](https://web.iconly.pro/) is the official icon library for NextUI, as mentioned in #1826. However, since Iconly is a paid service, you might consider using Solar Icons as an alternative. Below are some useful links:

- [GitHub Repository](https://github.com/480-Design/Solar-Icon-Set)
- [NPM Package](https://www.npmjs.com/package/solar-icon-set)

Additionally, Solar Icons can be used from Iconify at [iconify/solar](https://icon-sets.iconify.design/solar/), as they are required for [NextUI Pro](https://nextui.pro/documentation).
